src/components/Header/Header.js

```diff
@@ -158,6 +166,51 @@ class Header extends React.Component {
     this.setState({ logOutUrl: utils.renderDynamicLogOutLink(location) });
   }
 
+  /**
+   * handleEncoreLoggedInTimer(currentLocation, currentTime)
+   * This method is to set the timer to delete ‘PAT_LOGGED_IN’ cookie after its expiration time.
+   * This is to keep the logged in status consisitent with Encore server,
+   * so the patrons don’t have to log in when they are using non-account-requied operations,
+   * such as searching items.
+   * The default expiration time is 30 mins.
+   * @param {object} - The window.location object
+   * @param {number} - The milliseconds elapsed since January 1, 1970
```

Comment:
You could also add that it's the Date() time object, like the one above.

Reply:
But it is not an object. It is a string. I can say it is a string from Date.now().